batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/entity/Student.java

```diff
@@ -0,0 +1,60 @@
+package com.lftechnology.batch7crud.entity;
+
+/**
+ * Created by sanjay on 1/14/16.
+ */
+public class Student {
+    private int id;
+
+    private String firstName;
+    private String middleName;
+    private String lastName;
+    private String address;
+    private int grade;
+
+    public String getAddress() {
+        return address;
+    }
+
+    public void setAddress(String address) {
+        this.address = address;
+    }
+
+    public int getGrade() {
+        return grade;
+    }
+
+    public void setGrade(int grade) { this.grade = grade; }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)